frontend/src/routes/+layout.svelte

```diff
@@ -1,34 +1,38 @@
 <script lang="ts">
-  import { page } from '$app/stores';
-  import '$lib/app.postcss';
-  import { error, goesToErrorPage } from '$lib/error';
-  import UnexpectedErrorAlert from '$lib/error/UnexpectedErrorAlert.svelte';
-  import { onDestroy } from 'svelte';
-  import type { LayoutData } from './$types';
+import { page } from '$app/stores';
+import '$lib/app.postcss';
+import { error, goesToErrorPage } from '$lib/error';
+import UnexpectedErrorAlert from '$lib/error/UnexpectedErrorAlert.svelte';
+import { onDestroy } from 'svelte';
+import type { LayoutData } from './$types';
+import Notify from '$lib/notify/notify.svelte';
 
-  onDestroy(
-    page.subscribe((p) => {
-      error.set(p.error);
-    })
-  );
+onDestroy(
+  page.subscribe((p) => {
+    error.set(p.error);
+  })
+);
+
+// https://www.w3.org/TR/trace-context/#traceparent-header
+// so the page-load instrumentation can be correlated with the server load
+export let data: LayoutData;
+let notify: Notify;
```

Comment:
I don't think you're using this anymore are you? Now that you have global functions for creating notifications.
```suggestion
```